test/verify/check-networking-firewall

```diff
@@ -34,6 +34,12 @@ def wait_unit_state(machine, unit, state):
 
     wait(lambda: active_state(unit) == state, delay=0.2)
 
+def get_active_rules(machine):
```

Comment:
Most of this commit looks like it could go into a separate PR and be landed first, as it's some generalization? Then the "add fedora-31" main commit can just add the "fedora-31" special cases to the lists.